packages/web-app-files/src/helpers/share/role.ts

```diff
@@ -282,12 +289,13 @@ export abstract class LinkShareRoles {
     linkRoleUploaderFolder
   ]
 
-  static list(isFolder: boolean): ShareRole[] {
-    return this.all.filter((r) => r.folder === isFolder)
+  static list(isFolder: boolean, isOcis: boolean = false): ShareRole[] {
```

Comment:
Can you rename that usage of `isOcis` to a capability?

Reply:
I've changed to `capabilities.files_sharing?.public?.can_edit` is this ok?